### DISCUSSION TITLE
I want to use only the j and k keys and disable all other keys.

### DISCUSSION THREAD

**Discussion by yasainet** [philc/vimium#4480] [General]
Hello! Thank you for this amazing app.

As the title suggests, I would like to use only the j and k keys.
I want to disable all other keys.
This is for all sites.

Could you please tell me how to set this up?

I have a condition that makes my hands sweat a lot, so this app is essential for me!
I look forward to continuing to use it.

**Comment by seki-mei** [comment]
Add this to `Custom key mappings` in the plugin  options:
```
unmapAll
map k scrollUp
map j scrollDown
```

**Comment by yasainet** [comment]
Thank you very much!
The setup went smoothly, and I am enjoying using it!
